### ISSUE TITLE
Markdown - Emoji-Support

### ISSUE THREAD

**Issue by david-jointech** [BookStackApp/BookStack#929]
Emoji-Support with this syntax https://www.webpagefx.com/tools/emoji-cheat-sheet/ would be nice.

**Comment by itsalb3rt** [comment]
I am with you friend! 👍

**Comment by jjasghar** [comment]
It looks like https://www.bookstackapp.com/docs/admin/ut8mb4-support/ is possible, but I think that's just inputting the actual emoji, not the `:+1` or whatever.

**Comment by ssddanbrown** [comment]
@jjasghar Yeah, You should be able to already use emoji, Especially if you've created your instance in the last few years. We just don't support the specifically requested syntax, since I'm hesitant due to it not being a particular standard. I think both MacOS and Windows have built-in emoji pickers, Not so common in linux distros yet but there are apps or browser plugins available.

**Comment by jjasghar** [comment]
Ah nice, just as I thought. 

I understand the hesitation about the “:blah:” syntax/standard, but I think it’s safe to say it’s won out in emoji war. I don’t know anyone whoever thinks “(blah)” (the hipchat?) formatting anymore.

I know little to nothing about php, but someone has to have created a drop in converter right?

**Comment by ssddanbrown** [comment]
You could add something like this to the "Custom HTML Head Content" setting:

```html
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/emojify.js/1.1.0/css/basic/emojify.min.css" />
<script src="https://cdnjs.cloudflare.com/ajax/libs/emojify.js/1.1.0/js/emojify.min.js"></script>
<script>
window.addEventListener('DOMContentLoaded', () => {
    emojify.setConfig({img_dir: 'https://cdnjs.cloudflare.com/ajax/libs/emojify.js/1.1.0/images/basic/'});
    emojify.run(document.querySelector('.page-content'));
});
</script>
```

Note, this won't show them in the editor no provide any kind of "selector" interface. Also, the [library used](https://github.com/joypixels/emojify.js) is no longer maintained.

**Comment by delacor** [comment]
Any news on this ?

**Comment by ssddanbrown** [comment]
BookStack is now [managed on Codeberg](https://codeberg.org/bookstack/bookstack).

As part of this change, we're closing all existing GitHub issues and pull requests.
You'll instead find this issue on Codeberg here: 
https://codeberg.org/bookstack/bookstack/issues/929
  
Any further discussion for this will now be done on Codeberg.
Therefore this GitHub thread will be locked.
